src/server_manager/electron_app/write_environment_variables.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+#
+# Copyright 2020 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Should this fail if SENTRY_DSN is undefined?  We don't want to accidentally ship a release with no sentry key

Reply:
Done.